clients/src/main/java/org/apache/kafka/clients/consumer/CommitFailedException.java

```diff
@@ -32,4 +32,14 @@ public CommitFailedException(String message) {
         super(message);
     }
 
+    public CommitFailedException(Throwable t) {
```

Comment:
Any reason we need this constructor? In the past we haven't added constructors without a message, to help encourage more understandable exceptions.